client/src/pages/Mentees.jsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import styles from '../styles/Mentees.module.css';
+import * as api from '../api';
+// import MenteeImage from '../assets/images/empty_mentees.svg';
```

Comment:
Were you going to use this for when there are no mentees? I can also push this to a later sprint if u didn't intend to implement it!

Reply:
ya i had it show up when there's no mentees with this line:
`{(mentees.length === 0) && (<img src={MenteeImage} alt="mentees" />)}`

but it was kinda glitchy; so whenever you refresh the mentees page the image would appear for a split second before the mentee cards are displayed. wasn't sure if that was an issue or not, so i just ultimately left it out